core/src/main/java/io/undertow/server/handlers/cache/LRUCache.java

```diff
@@ -172,6 +172,18 @@ public void clear() {
         accessQueue.clear();
     }
 
+    public Set<K> keys(){
```

Comment:
There's a keySet() method defined already serving the same purpose.
Further more it properly makes returned keys read only. 
We should remove this keys() method.